app/routes/users/components/StudentConfirmation.tsx

```diff
@@ -80,13 +80,13 @@ const StudentConfirmation = ({
   return (
     <Container>
       <div>
-        <h2>Verifiser studentepost</h2>
+        <h2>Verifiser studente-post</h2>
```

Comment:
```suggestion
        <h2>Verifiser student e-post</h2>
```
Imo. this looks better with a space, but I'm happy as long as we are consistent about it

Reply:
I've now found examples of OsloMet, Kristiania and UiB writing "student-e-post" with a hyphen, so I'll go with that. Nice catch